master/static/srv/entrypoint.sh

```diff
@@ -7,14 +7,6 @@ set -e
 
 STARTUP_HOOK="startup-hook.sh"
```

Comment:
i'm wondering if moving the python check before startup hooks will mess with anyone..

Reply:
@stoksc these checks were running prior to startup hook for slurm; however, not for other cases. 

Are we wanting to setup task logging after the startup hook are set up?